include/oneapi/dpl/internal/serial_algorithm_impl.h

```diff
@@ -38,10 +37,9 @@ for_each_n(_InputIterator __first, _Size __n, _Function __f)
     return __first;
 }
 
-#    else
+#else
 using ::std::for_each_n;
```

Comment:
Is there any C++ library with doesn't have `::std::for_each_n` and that library is compiled with "std=C++17"?
If there is not, that I think that we should remove 26-42 lines as well.